plugins/hypervisors/xenserver/src/main/java/org/apache/cloudstack/hypervisor/xenserver/ExtraConfigurationUtility.java

```diff
@@ -101,6 +109,7 @@ private static void applyConfigWithKeyValue(Connection conn, VM vm, Map<String,
         }
 
         try {
+            // param with '_'
```

Comment:
do we need this comment? it is a bit unclear if this is enforced or a contextual condition or ...

Reply:
just to keep as note. these params are with underscore, as the hypens were replaced with underscores.